src/explore-education-statistics-common/src/lib/dom/findAllForId.ts

```diff
@@ -0,0 +1,13 @@
+/**
+ * Find elements that match the given
+ * {@param id} and a {@param selector}.
+ */
+export default function findAllByText<T extends HTMLElement = HTMLElement>(
```

Comment:
Remove this completely as `querySelectorAll` achieves the same thing

Reply:
done